README.md

```diff
@@ -107,6 +107,7 @@
 - [BorgBackup](https://github.com/borgbackup/borg) - A fork of [Attic](https://attic-backup.org) deduplicating backup program written in Python.
 - [Burp](http://burp.grke.org/) - Network backup and restore program.
 - [DREBS](https://github.com/dojo4/drebs) - AWS EBS backup script that supports strategies.
+- [Dar](http://dar.linux.free.fr) - Robust archiving tool in the style of tar, with tons of functionality.
```

Comment:
- Missing source code link
- Missing license and language/platform tags

(please check formatting guidelines above)

Also since we're trying to update contribution guidelines/clarify _awesomeness_ criteria (https://github.com/awesome-foss/awesome-sysadmin/pull/357) can you please try to answer the following questions as best you can:

- **Have you used it? For how long?**
- **Is this in a personal or professional setup?**
- **How many devices/users/services/... do you manage with it?**
- **Biggest pros/cons compared to other solutions?**
- **Any other comments about your use case, things you've found excellent, limitations you've encountered... ?**

Thanks

Reply:
Hi @nodiscc,

Initially I did provide provide the github source link, but decided to provide the other link as that better explains what 'dar' is to a visitor.

**Have you used it? For how long?**
I have previously used 'dar' - switched to something else (rsync).
I have now used 'dar' for around 2 years for taking backups of all my valuable files (photos, videos, various documents). To that end I have built a [set of scripts](https://github.com/per2jensen/dar-backup) that uses 'dar' and 'parchive2'.  'dar' has proven to be a very good solution for my use case (backup regularly to file server, from file server to different usb disks) - it is easy to verify archive integrity once the archive has been moved.

**Is this in a personal or professional setup?**
This is a personal setup - I am working on self hosting more and more, and especially degoogling.

**How many devices/users/services/... do you manage with it?**
2 workstations, 1 server + backup of my cloud storage (which keep all mobile phone photos)

**Biggest pros/cons compared to other solutions?**
'dar' has been around for more than 20 years, it is still actively developed and maintained by the same person. The maintainer is quite responsive to bug reports and works with reporters to either fix an issue or develop a solution when time permits and it fits into 'dar''s mission.

It is like tar on stereoids:  strong feature set, very robust and easy to script. I especially like the dar_manager which can build a catalog of all files in a set of archives, and help restoring the correct version of a file, that might be in multiple archives. I like very much that I can put a static binary along the archives, and it will be easy to do restores in the future, no need to install anything, just follow a guide.

Currently I am working on my backup scripts to make use of dar_manager.

The con obviously is that it is a single person responsible for the development of 'dar'. However, the last 20 years has shown deep commitment, so I don't think it should weigh too much.

**Any other comments about your use case, things you've found excellent, limitations you've encountered... ?**
Excellent documentation is available as well as a list of [limitations](http://dar.linux.free.fr/doc/Limitations.html).

Limitations seen from a user perspective is that 'dar' is not widely used (judging by number of github projects, blogs, etc...), so if the user has a problem, some self study is needed to figure out a solution.

Reply:
I updated the oroginal dar entry description to:

- [Dar](http://dar.linux.free.fr/) - dar which stands for Disk ARchive, is a robust and rich featured archiving and backup software of the tar style. [Source Code](https://github.com/Edrusb/DAR), `GPL-2.0` `C++`

Reply:
Thanks for the useful review, and for updating the entry to follow the new syntax.